src/hardware/vga_paradise.cpp

```diff
@@ -36,7 +36,7 @@ typedef struct {
 
 	Bitu clockFreq[4];
 	Bitu biosMode;
-} SVGA_PVGA1A_DATA;
+} ;
```

Comment:
Remove space between the closing bracket and semicolon (it will bring this code back to our formatting standard).